docs/en/sql-reference/statements/show.md

```diff
@@ -15,12 +15,91 @@ Returns a single `String`-type ‘statement’ column, which contains a single v
 
 ## SHOW DATABASES {#show-databases}
 
+Prints a list of all databases. 
+
+Syntax:
+
+```sql
+SHOW DATABASES [LIKE '<pattern>'] [ILIKE '<pattern>'] [NOT LIKE '<pattern>'] [LIMIT <N>] [INTO OUTFILE filename] [FORMAT format]
+```
+
+This statement is identical to the query:
+
+```sql
+SELECT name FROM system.databases [WHERE name LIKE '<pattern>'] [WHERE name ILIKE '<pattern>'] [WHERE name NOT LIKE '<pattern>'] [LIMIT <N>] [INTO OUTFILE filename] [FORMAT format]
+```
+
+### SHOW DATABASES LIKE {#show-databases-like}
```

Comment:
I think it's better to place all the examples under the header `### Examples`.
Each example doesn't need the header. You can preface examples with a description of what the example does. For example, "Getting database names, containing the symbol sequence `de` in their names."